source/projects/webRadio/de/index.md

```diff
@@ -0,0 +1,136 @@
+
+## Die Idee
+In Zeiten der zunehmenden Automation wollten wir einmal mehr hinter die Kulissen blicken und ein bluetooth-fähiges Radio mit Hilfe eines Raspberry Pi 3 und einem **Autostart-Programm** zu einem Autoplay Web Radio hacken. Die Kommunikation vom Laptop zum RasPi erfolgt dabei über SSH - vom RasPi zum Radio über Bluetooth. Die gesamte Konfiguration (Bluetooth, VLC-Mediaplayer Command, Delays etc.) wurde in ein **Shell-Skript** geschrieben, welches schließlich mit der Autostart-Datei des Raspberry's verknüpft wurde. 
+
+## Die Hard- und Software
+Hardwareseitig ist dieses Projekt sehr leicht zu realisieren, da man lediglich:
+* einen Laptop
+* einen Raspberry 3 und
+* ein Bluetooth-fähiges Radio
+
+benötigt. Die Anleitung funktioniert natürlich auch mit anderen Einplatinencomputern unter Linux – solange diese ein boardinternes Chipset für die WLAN- und Bluetooth-Funktion verbaut haben.
+
+Damit der Laptop mit dem Raspberry kommunizieren kann, muss zunächst einmal ein Betriebssysteme auf den RasPi aufgesetzt werden – in unserem Falle Rasbian. Nach erfolgreicher Installation des Betriebssystems kann mit der Konfiguration des Raspberry's begonnen werden. Je nachdem ob der RasPi headless (via SSH) oder mit head (via Tastatur und Monitor) betrieben werden soll, dient das **Netzwerkprotokoll Secure Shell (SSH) als Kommunikationsschnittstelle** zu unserem RasPi. In unserem Tutorial wurde die headless Inbetriebnahme unter Nutzung von SSH beschrieben und darüber hinaus das kryptografische Verschlüsselungsverfahren der SSH beleuchtet.
+
+{% include macro-image-section-markdown.html src="../images/webRadio_network.jpeg" caption="Schnittstellen zwischen den einzelnen Devices (©eigene Darstellung, Technologiestiftung Berlin) "%}
+
+## RasPi mit dem Radio verbinden
+Die meisten von uns verbinden sich tagtäglich via Bluetooth mit anderen Devices – ganz einfach unter Nutzung des Graphic User Interface (GUI) mit Tastatur und Maus. Das ist in der Regel auch schneller, doch in unserem Workshop stellen wir die Verbindung lediglich mit ein paar Terminal-Command her, denn genau hier liegt die **Magie der Automation**. Speichert man die Commands in einem Shell-Skript und bindet dieses kleine Skript in den Autostart des Raspberry's ein, verbindet sich der Raspberry beim nächsten Hochfahren automagically mit dem Bluetooth-Radio. 
+
+Zum Verbinden mit dem Radio sind folgenden Befehle notwendig, wobei die angegebene MAC-Adresse des Radios durch die des eigenen Radios ausgetauscht werden muss (Kommentare werden nicht im Terminal angezeigt):
+```shell
+$ bluetoothctl
+> Agent registered
+
+$ [bluetooth]# power on
+> changing power on succeeded
+
+$ [bluetooth]# default-agent
```

Comment:
I would suggest not to add the `# comment` in the examples behind the lines. Add it before or split the code blocks and write the comments in between

Reply:
actually, the `#` comes together with bluetoothctl. It is not meant as a comment. eg. `[bluetoothctl]# ` comes directly out of command line and `power on` is the actual command